frontend/class-breadcrumbs.php

```diff
@@ -1012,7 +1024,7 @@ private function wrap_breadcrumb() {
 			$output = apply_filters( 'wpseo_breadcrumb_output', $output );
 
 			if ( WPSEO_Options::get( 'breadcrumbs-prefix' ) !== '' ) {
-				$output = "\t" . WPSEO_Options::get( 'breadcrumbs-prefix' ) . "\n" . $output;
+				$output = "\t" . '<span class="breadcrumbs-prefix">' . WPSEO_Options::get( 'breadcrumbs-prefix' ) . "</span>\n" . $output;
```

Comment:
Can you explain the reason for adding the `class`?

Reply:
Give the opportunity to style it. Previously, this was just text thrown in the middle of nowhere, without even a wrapper element.